Rakefile

```diff
@@ -1,13 +1,30 @@
+# frozen_string_literal: true
+
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new(:rubocop) do |task|
+  # These make the rubocop experience maybe slightly less terrible
+  task.options = ['-D', '-S', '-E']
+
+  # Use Rubocop's Github Actions formatter if possible
+  if ENV['GITHUB_ACTIONS'] == 'true'
+    rubocop_spec = Gem::Specification.find_by_name('rubocop')
+    task.formatters << 'github' if Gem::Version.new(rubocop_spec.version) >= Gem::Version.new('1.2')
+  end
+end
+
+task default: [:rubocop]
+
 begin
   require 'rubygems'
   require 'github_changelog_generator/task'
-rescue LoadError
-else
```

Comment:
Is it needed to change this style? You can place a comment in there as well. Essentially what it says is "this part can fail but the RakeTask can't fail with a `LoadError`".

Reply:
I copied it from https://github.com/voxpupuli/puppet-lint_modulesync_configs/blob/master/moduleroot/Rakefile.erb#L7. We can keep the current style if you like, but we need at least a comment in the rescue section or raise a warning. I think keeping it silent is okay since we do that since months.